### PR TITLE
Travel fund request for @mihaiep;  tag @nodejs/community-committee

### PR DESCRIPTION
**Pull request by mihaiep**
Travel fund request for @mihaiep at Index 2018 for Feb 20;

### CODE REVIEW

**Comment by mhdawson** [comment]
Will be good to have Mihai Ene-Pietrosanu as another member of the user feedback team at the community day for the user feedback session.

**Comment by MylesBorins** [comment]
@mihaiep have you registered as an individual member of the foundation? That is a requirement for the travel fund. You can find instructions at https://github.com/nodejs/members/issues/1 

Please let me know if you have any issues

**Comment by mihaiep** [comment]
@MylesBorins thank you for pointing this. Registered as an individual member (Membership Order 311096). The discount coupon is not working. thank you

**Comment by joyeecheung** [comment]
@mihaiep Hi, there is a conflict in this document, can you rebase? There have been no objections in 18 days so I'll go ahead and merge this when the conflict is resolved.

**Comment by mihaiep** [comment]
@joyeecheung Hi, I'm in a process to get better on github , not sure what you mean. thanks for patience

File: MEMBER_TRAVEL_FUND.md [hunk 1]
```diff
@@ -61,7 +61,10 @@ Bryan English | JSConf.cn 2017 | Code & Learn Mentor | Shanghai, CN | 15 July -
 Tobias Nießen | NINA 2017 | Collaborator Summit and Code & Learn | Vancouver, BC, CA | 5 Oct - 8 Oct 2017 | €1300
 Tiancheng Gu | NINA 2017 | Collaborator Summit | Vancouver, BC, CA | 3 Oct – 8 Oct 2017 | US$ 1200
 Ruben Bridgewater | NINA 2017 | Collaborator Summit and Code & Learn | Vancouver, BC, CA | 4 Oct - 8 Oct 2017 | €1500
+Joyee Cheung | Diagnostics WG Summit | Attendance | Ottawa, ON, CA | 12 Feb - 13 Feb 2018 | US$ 1400
```
Comment (joyeecheung) [inline MEMBER_TRAVEL_FUND.md]: There seems to be something wrong with the merge, can you remove this line?

Comment (mihaiep) [reply MEMBER_TRAVEL_FUND.md]: @joyeecheung could you check now? thanks

**Pull request merged**